cluster/common.sh

```diff
@@ -963,6 +969,31 @@ function create-certs {
   KUBECFG_KEY_BASE64=$(cat "${CERT_DIR}/pki/private/kubecfg.key" | base64 | tr -d '\r\n')
   KUBEAPISERVER_CERT_BASE64=$(cat "${CERT_DIR}/pki/issued/kube-apiserver.crt" | base64 | tr -d '\r\n')
   KUBEAPISERVER_KEY_BASE64=$(cat "${CERT_DIR}/pki/private/kube-apiserver.key" | base64 | tr -d '\r\n')
+
+  AGGR_CERT_DIR="${KUBE_TEMP}/easy-rsa-master/aggregator"
```

Comment:
I set them up in a different directory because otherwise I was getting the same CA and the goal here is was to have the proxy certs signed by a different CA than the rest of our certs.